lib/ims/lti/services/message_authenticator.rb

```diff
@@ -33,8 +33,7 @@ def simple_oauth_header
           @options.merge(
             {
               consumer_key: consumer_key,
-              consumer_secret: @secret,
-              callback: 'about:blank'
```

Comment:
I don't know why we added it initially, we do parse the parameters passed into us, and it should automatically get added from there if it's part of the message. Otherwise it won't but us adding it is basically us enforcing that it's being used. Doesn't really benefit us much besides saying we implement oauth 1.0a exactly, but we don't use that piece of the functionality in the spec. I'm in favor of removing it.